input/cql/OncologyPainIntensityQuantifiedFHIR.cql

```diff
@@ -38,6 +42,9 @@ define "Chemotherapy Within 31 Days Prior and During Measurement Period":
     where ChemoAdministration.performed.toInterval ( ) during Interval[start of "Measurement Period" - 31 days, end of "Measurement Period"]
       and ChemoAdministration.status = 'completed'
 
+/*
+@QICore6Uplift: Update Condition to ConditionProblemsHealthConcerns
+*/ 
```

Comment:
As written this should also be a union of ConditionProblemsHealthConcerns and ConditionEncounterDiagnosis